fs/move.ts

```diff
@@ -30,6 +31,14 @@ export async function move(
     );
   }
 
+  const srcAbsolute = src instanceof URL ? src.pathname : resolve(src)
+  const destAbsolute = dest instanceof URL ? dest.pathname : resolve(dest)
+
+  if (srcAbsolute === destAbsolute) {
```

Comment:
Can this check made into a common util in `fs/_util.ts`? (like `isSubdir`)

Reply:
Will do.